src/edu/gatech/cs2340/trydent/ContinuousEvent.java

```diff
@@ -0,0 +1,59 @@
+package edu.gatech.cs2340.trydent;
+
+/**
+ * Represents an event that is called by the engine every frame.
+ * @author Garrett Malmquist
+ *
+ */
+public abstract class ContinuousEvent {
+	
+	private boolean started = false;
+	
+	public ContinuousEvent() {
+		TrydentEngine.addContinuousEvent(this);
+	}
+	
+	/**
+	 * Called by the engine on update.
+	 */
+	protected final void doUpdate() {
```

Comment:
I can see users wanting to add Timer-like events what are contained within Trydent - leaving this open to subclassing is fine by me.